testing/unit_tests/cases_loader.py

```diff
@@ -0,0 +1,242 @@
+        LoaderParamsKW.ROI_PARAMS: {"suffix": None, "slice_filter_roi": None},
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": ["ct"]}
+    }
+
+    return loader_parameters
+
+
+def case_dropout_input_2_5_5():
+    seg_pair = {"input": torch.rand((2, 5, 5))}
+    return seg_pair
+
+
+def case_dropout_input_1_5_5():
+    seg_pair = {"input": torch.rand((1, 5, 5))}
+    return seg_pair
+
+
+def case_dropout_input_5_5_5_5():
+    seg_pair = {"input": torch.rand((5, 5, 5, 5))}
+    return seg_pair
+
+
+def case_dropout_input_5_5_5_3():
+    seg_pair = {"input": (torch.rand((5, 5, 5, 3)) * torch.tensor([1, 0, 1], dtype=torch.float)).transpose(0, -1)}
+    return seg_pair
+
+
+def case_dropout_input_7_7_4():
+    seg_pair = {"input": (torch.rand((7, 7, 4)) * torch.tensor([1, 0, 0, 0], dtype=torch.float)).transpose(0, -1)}
+    return seg_pair
+
+
+def case_load_dataset_2d_png():
+    loader_parameters = get_microscopy_loader_parameters()
+    loader_parameters.update({
+        LoaderParamsKW.TARGET_SUFFIX: ["_seg-myelin-manual"],
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": [], "balance": {}},
+        LoaderParamsKW.SLICE_AXIS: "axial",
+        LoaderParamsKW.SLICE_FILTER_PARAMS: {"filter_empty_mask": False, "filter_empty_input": True},
+        LoaderParamsKW.PATCH_FILTER_PARAMS: {"filter_empty_mask": False, "filter_empty_input": False},
+        LoaderParamsKW.MULTICHANNEL: False
+    })
+
+    model_parameters = {
+        ModelParamsKW.NAME: "Unet",
+        ModelParamsKW.DROPOUT_RATE: 0.3,
+        ModelParamsKW.BN_MOMENTUM: 0.1,
+        ModelParamsKW.FINAL_ACTIVATION: "sigmoid",
+        ModelParamsKW.DEPTH: 3
+    }
+
+    transform_parameters = {
+        TransformationKW.NUMPY_TO_TENSOR: {},
+    }
+
+    return loader_parameters, model_parameters, transform_parameters
+
+
+def case_2d_patches_and_resampling():
+    loader_parameters = get_microscopy_loader_parameters()
+    loader_parameters.update({
+        LoaderParamsKW.TARGET_SUFFIX: ["_seg-myelin-manual"],
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": [], "balance": {}},
+        LoaderParamsKW.SLICE_AXIS: "axial",
+        LoaderParamsKW.SLICE_FILTER_PARAMS: {"filter_empty_mask": False, "filter_empty_input": True},
+        LoaderParamsKW.PATCH_FILTER_PARAMS: {"filter_empty_mask": False, "filter_empty_input": False},
+        LoaderParamsKW.MULTICHANNEL: False
+    })
+
+    model_parameters = {
+        ModelParamsKW.NAME: "Unet",
+        ModelParamsKW.DROPOUT_RATE: 0.3,
+        ModelParamsKW.BN_MOMENTUM: 0.1,
+        ModelParamsKW.FINAL_ACTIVATION: "sigmoid",
+        ModelParamsKW.DEPTH: 3,
+        ModelParamsKW.LENGTH_2D: [256, 128],
+        ModelParamsKW.STRIDE_2D: [244, 116]
+    }
+
+    transform_parameters = {
+        TransformationKW.RESAMPLE: {
+            TransformationKW.W_SPACE: 0.0002,
+            TransformationKW.H_SPACE: 0.0001
+        },
+        TransformationKW.NUMPY_TO_TENSOR: {},
+    }
```

Comment:
Notice this pattern where you have to redefine these two dicts over and over again? It is probably safer to overwrite a default one solely on the modifications. 

Again, best to test it out as the very default value can introduce more aberrations. 

Set/Empty accordingly. 

Another benefit of a single default and overwrite/empty in unit is the advantage that such alterations are IMMEDIATELY visible and apparent for new coders joining, otherwise, a DIFF is required to really tell what changed. All part of ability to improve readability/ease of interpretation.